CHANGES.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 1.9.10 (core) / 0.25.10 (libraries)
+
+### New
+
+- Implemented the `AutomationCondition.replace()` function to improve ergonomics of updating sub-conditions.
```

Comment:
Added a new `.replace()` method to `AutomationCondition`, which allows sub-conditions to be modified in-place.